### PR TITLE
Add Twitter button to Portal Navbar

### PR DESCRIPTION
**Pull request by jukent**
Closes https://github.com/ProjectPythia/projectpythia.github.io/issues/136

### CODE REVIEW

**Comment by jukent** [comment]
I tried a few ways to push the navbar Twitter button to the right but am having no luck. I would appreciate another set of eyes on the html, as well as style opinions. Also, maybe it's not that important to have the button on the right?

File: sphinx_pythia_theme/header.html [hunk 1]
```diff
@@ -33,6 +33,11 @@
           <a class="nav-link" href="/about.html">About</a>
         </li>
       </ul>
+      <ul class="nav navbar-nav navbar-right">
```
Comment (kmpaul) [inline sphinx_pythia_theme/header.html:36]:
```suggestion
      <ul class="navbar-nav ms-auto">
```

**Review by kmpaul — CHANGES_REQUESTED** [review]

**Comment by kmpaul** [comment]
@jukent:  This is Bootstrap 5, so you need to make sure you are reading the documentation for the correct version of Bootstrap.

I fixed the right-align issue, but I think the button seems out of place and doesn't match the theme.  Can we change the button to something that looks better with the theme?

**Review by mgrover1 — APPROVED** [review]
It looks great! Thanks @jukent for adding this

**Comment by jukent** [comment]
@kmpaul Is there something significantly different between this portal sight and the Xdev blog? I would like to copy the way the Twitter button looks over there, but cannot seem to replicate it. Both are Bootstrap 5 correct?

**Comment by jukent** [comment]
That makes sense then.

**Comment by jukent** [comment]
Build failure with "Error: Unable to locate executable file: /usr/share/miniconda/condabin/mamba. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable." I don't think this is related to this small change.

**Review by kmpaul — APPROVED** [review]
This looks good!

**Comment by kmpaul** [comment]
@mgrover1: This is read for review, again.

**Review by mgrover1 — APPROVED** [review]
Looks great - thanks @jukent ! 👍

**Pull request merged**